corehq/util/model_log.py

```diff
@@ -2,14 +2,14 @@
 from django.utils.encoding import force_text
 
 
-def log_model_change(user, model_object, message=None, fields_changed=None, is_create=False):
+def log_model_change(user, model_object, message=None, fields_changed=None, is_create=False, is_delete=False):
```

Comment:
should validate that `is_created` and `is_deleted` aren't both True. At this point it may also be worth just passing in the flag in directly. You could also alias them in this file to make it clearer what the options are:

```python
class ModelAction(Enum):
    ADDITION = ADDITION
    CHANGE = CHANGE
    DELETION = DELETION

def log_model_change(user, model_object, message=None, fields_changed=None, action: ModelAction = ModelAction.CHANGE):
   ...
   action_flag = action.value
   ...

Reply:
Picked this suggestion with slight change to naming to keep it more relatable.

Skipped type check. I have not wrapped my head around it completely yet and has not become a habit. Unless that is a blocker.